ProjFS.Mac/PrjFSKext/ProviderMessaging.cpp

```diff
@@ -180,10 +188,22 @@ bool ProviderMessaging_TrySendRequestAndWaitForResponse(
         }
         else
         {
+            int loopCount = 0;
             while (!message.receivedResult &&
                    !s_isShuttingDown)
             {
+                if (loopCount > 12)
+                {
+                    KextLog_Error("NO RESPONSE AFTER %d seconds Count: Count:%lld Processed:%lld messageType:%d", loopCount * 5, nextMessageId, s_messagesSent, messageType);
+                }
+                
+                if (s_nextMessageId - s_messagesSent > 1000)
+                {
+                    KextLog_Error("While Loop Message Count: Count:%lld Processed:%lld messageType:%d", s_nextMessageId, s_messagesSent, messageType);
```

Comment:
In your testing so far have you seen this message get logged?

Reply:
Yes, I believe I saw that Friday.